tools/civic/civic_qrt.xml

```diff
@@ -0,0 +1,82 @@
+<tool name="CIViC Query Report Tool" id="civic_qrt" version="0.1">
+  <description>CIViC Query Report Tool reads a MAF file, calls CIViC for each line therein, and outputs a table in HTML format for easy inclusion into downstream reports</description>
+  <command interpreter="python">
+   civic_qrt.py
+   -i "${input_file}"
+   -o "${out_file}"
+   -c "${host}"
+   -col "${entrez_id_column}"
+  </command>
+  <inputs>
+    <param name="input_file" type="data" format="maf" 
+      label="MAF File from Oncotator" help="If empty, upload or import a MAF dataset."/>
```

Comment:
Is this tool supposed to run at the remote site or central site? The "input_file" should be located at the appropriate site.